### DISCUSSION TITLE
How to use the Time Range filter of dashboard and pass to API using ISO format

### DISCUSSION THREAD

**Discussion by possebon** [grafana/grafana-infinity-datasource#642] [Q&A] [answered]
Hi all,

I'm new to the Infinity datasource and I'm using to consume data from a JSON API.

I'm using a POST request, and informing as `x-www-form-urlencoded` parameters (did not work sending as JSON Body). The API endpoint need to receive two parameters:

- dateFrom as 1991-08-30T14:42:53.522Z
- dateTo as 1991-11-15T22:48:59.347Z

I could not find a way to send it.

Any help would be very appreciated.

Kind regards,

**Comment by yesoreyeram** [comment, marked as answer]
You may need to pass grafana global variables in a format you want 

https://grafana.com/docs/grafana/v8.5/variables/variable-types/global-variables/#__from-and-__to

**Reply by possebon** [reply]
Worked perfectly!! Thanks!
